plugin-server/src/config/config.ts

```diff
@@ -209,19 +209,6 @@ export function getDefaultConfig(): PluginsServerConfig {
     }
 }
 
-export const sessionRecordingConsumerConfig = (config: PluginsServerConfig): PluginsServerConfig => {
-    // When running the blob consumer we override a bunch of settings to use the session recording ones if available
-    return {
-        ...config,
-        KAFKA_HOSTS: config.SESSION_RECORDING_KAFKA_HOSTS || config.KAFKA_HOSTS,
-        KAFKA_SECURITY_PROTOCOL: config.SESSION_RECORDING_KAFKA_SECURITY_PROTOCOL || config.KAFKA_SECURITY_PROTOCOL,
-        POSTHOG_REDIS_HOST:
-            config.POSTHOG_SESSION_RECORDING_REDIS_HOST || config.INGESTION_REDIS_HOST || config.POSTHOG_REDIS_HOST,
```

Comment:
since this was set on the running pods how did we end up falling through to the incorrect host?

Reply:
i guess we were passing in an unexpected config?

Reply:
No this code just wasn't ever used by the realtime-manager - that was the core issue, hence reworking it to be super explicit.

Reply:
🙈